kubevirt-ami-centos.json

```diff
@@ -38,6 +39,11 @@
       "source": "image-files/virtctl",
       "destination": "/home/centos/"
     },
+    {
+      "type": "file",
+      "source": "tests",
+      "destination": "/home/centos"
+    },
```

Comment:
The test playbook uploads the test. It is a duplicate step. But eventually I think we will move it here to make the tests/manifests a permanent part of the image.